src/styles.scss

```diff
@@ -1211,6 +1196,11 @@ $truncate-text-values: 1, 2, 3;
   padding-right: 1.6rem;
 }
 
+// This style should be used for the new mat-card designs that have reduced padding on bottom of card, 8px
+mat-card.mat-card-padding {
+  padding-bottom: 8px;
+}
```

Comment:
have you tried not creating a new class for this and just using one of the bootstrap pb classes?

Reply:
mm I didn't try that, I can use pb-3 it gives me 10px if that's good with you?
![image](https://user-images.githubusercontent.com/97123241/152401986-26a1dbe2-a0b1-4b9d-aeda-32e7bfe0ee57.png)

Reply:
fine with me